packages/react-client/src/ReactFlightClientHostConfigNoStream.js

```diff
@@ -12,19 +12,22 @@ export type StringDecoder = void;
 export const supportsBinaryStreams = false;
 
 export function createStringDecoder(): void {
+  // eslint-disable-next-line react-internal/prod-error-codes
```

Comment:
What about defining something like an `NotImplementedError` or `UnminifiedError` which is automatically ignored by the lint rule and parser?